andela/ah-infinity-stones-frontend
# #163590086 Implement User Logout

#### What does this PR do?
- Implement user logout

#### Description of Task to be completed?
- Convert Navbar to class component
- Add onclick method to logout link

#### How should this be manually tested?
1. `$ git clone -b ft-logout-163590086 https://github.com/andela/ah-infinity-stones-frontend.git`  and `$ cd ah-infinity-stones-frontend`
2. Install the dependencies - `$ npm install`
3. Start the app by running `$ npm start` - This will run the app in the development mode.  
 To test the feature. Open http://localhost:3000/login  to view it in the browser. Provide email and password and click submit. This assumes that you have visited `http://localhost:3000/signup`, created and activated your account. This will redirect you to the homepage with the logout link.
4. Click logout and all your auth details will be cleared, taking u back to the homepage.
4. Test the app by running `$ npm test`
5. Check test coverage by running `$ npm test -- --coverage`

#### What are the relevant pivotal tracker stories?
[#163590086](https://www.pivotaltracker.com/story/show/163590086)
